content/blog/global-case-study.md

```diff
@@ -16,76 +16,82 @@ A leading comprehensive global mental health solution for employers. It started
 
-The next-generation tech-enabled precision mental healthcare company went international, localizing the experience and bringing quality care to 41 countries in 22 new languages. While it continues to extend the care globally, let's take a look at the technical how.
+The next-generation tech-enabled precision mental healthcare company went international, localizing the experience and bringing quality care to 41 countries in 22 new languages.
+
+To extend the care globally, they had to solve some challenges, which included internationalization of the application along with content localization.
 
 ## What We Did
 
-- Created a framework to enable global offerings in the web application
-- Country specific language content
-- Streamline workflow integrating a third-party translation service~
-- Extended international offering to 40 more countries in 9 more languages
-- Upgraded major project dependencies to access internationalization packages
+- Created a framework to globalize care services
+- Provide country specific localized content
+- Streamline and automate translation workflow, integrating a third-party translation service
+- Extended international offering to 41 more countries in 22 more languages
 
 ## Outcomes
 
-- 40+ countries
-- 9+ languages
-- 3 months for going live
-- Easy quick process to add more countries and languages
+- Providing care for 41+ countries in 22+ languages
+- Ability to introduce the product to a new country within a week
+- Having first class support for right-to-left languages
+- Localize content based on country
 
 ## Challenges and Solutions
 
-### Tech Debt
-
-In the JavaScript world, it doesn't take long for dependencies to become stale. While relatively older versions of most of the major libraries are stable, we cannot leverage the new and improved features that the newer versions provide to help us complete our job easily. In certain cases, it was mandatory to reach to a minimum version requirement to be able to add internationalization libraries.  We learnt this the hard way when we were initially trying to add internationalization changes to the codebase. Thus, as our first step, we identified the necessary libraries that required upgrades, resulting into a chain of few other dependent libraries as well, and spent few sprints solely to take care of it. This also led us to recognize several other dependencies that we could benefit from upgrading and helped plan the process to tackle this later on, which deserves its own separate story.
-
-### Internationalization
+### UI Translations
 
-#### Translations
+**Challenge**: Translating the UI based on user's language preference
 
-We had translatable content stored both on the frontend and backend, each of which had their own challenges.
+**Solution**:
 
-#### Frontend
+- Identifying and extracting string literals to JSON locale files
+- Update existing usages of string literals with localized versions
+- Introduce third-party translation provider Crowdin
```

Comment:
make crowding bold. @ArohiParikh @rushaliparikh @2KAbhishek @utsav00 we need one of you to take ownership to work with Crowdin. Can we make it happen so that they also promote this case study on their website, LinkedIn etc?